src/BSON/PersistableEnum.stub.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @generate-class-entries static
+ * @generate-function-entries
+ */
+
+namespace MongoDB\BSON;
+
+trait PersistableEnum
```

Comment:
Seeing these changes makes me appreciate the work people put into gen_stub.php for PHP core - it's just so much easier to define classes now than it was before when everything had to be done manually...

Reply:
![](https://media.giphy.com/media/5wWf7GR2nhgamhRnEuA/giphy.gif)